src/node/tx_receipt.h

```diff
@@ -3,15 +3,15 @@
 #pragma once
 
 #include "ccf/receipt.h"
-#include "node/history.h"
+#include "ccf/tx_receipt_path.h"
 
 namespace ccf
 {
   struct TxReceipt
```

Comment:
Is it necessary that we keep this class/file? Looks like a copy of the public one to me.

Reply:
Good spot, this should have been deleted.